src/current/_includes/molt/molt-troubleshooting-replication.md

```diff
@@ -0,0 +1,257 @@
+
+##### Wrong replication slot name
+
+~~~
+run SELECT pg_create_logical_replication_slot('molt_slot', 'pgoutput'); in source database
+~~~
+
+**Resolution:** {% if page.name != "migrate-load-replicate.md" %}[Create the replication slot]({% link molt/migrate-load-replicate.md %}#configure-source-database-for-replication){% else %}[Create the replication slot](#configure-source-database-for-replication){% endif %} or verify the correct slot name:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT pg_create_logical_replication_slot('molt_slot', 'pgoutput');
+~~~
+</section>
+
+{% if page.name == "migrate-resume-replication.md" %}
+##### Resuming from stale location
+
+<section class="filter-content" markdown="1" data-scope="postgres">
+For PostgreSQL, the replication slot on the source database tracks progress automatically. Clearing the memo table is only necessary if the replication slot was destroyed and you need to restart replication from a specific LSN.
+
+**Resolution:** Clear the `_replicator.memo` table:
+</section>
+
+<section class="filter-content" markdown="1" data-scope="mysql">
+**Resolution:** Clear the `_replicator.memo` table to remove stale GTID checkpoints:
+</section>
+
+<section class="filter-content" markdown="1" data-scope="oracle">
+**Resolution:** Clear the `_replicator.memo` table to remove stale SCN (System Change Number) checkpoints:
+</section>
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+DELETE FROM _replicator.memo WHERE true;
+~~~
+{% endif %}
+
+<section class="filter-content" markdown="1" data-scope="mysql">
+##### Repeated binlog syncing restarts
+
+If Replicator repeatedly restarts binlog syncing or starts replication from an unexpectedly old location, this indicates an invalid or purged GTID. When an invalid GTID is provided, the binlog syncer will fall back to the first valid GTID.
+
+**Resolution:** Verify the GTID set is valid and **not** purged:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+-- Check if GTID is in executed set
+SELECT GTID_SUBSET('your-gtid-set', @@GLOBAL.gtid_executed) AS in_executed;
+
+-- Check if GTID is purged
+SELECT GTID_SUBSET('your-gtid-set', @@GLOBAL.gtid_purged) AS in_purged;
+~~~
+
+Interpret the results as follows:
+
+- If `in_executed` returns `1` and `in_purged` returns `0`, the GTID is valid for replication.
+- If `in_purged` returns `1`, the GTID has been purged and you must find a newer consistent point.
+- If both return `0`, the GTID doesn't exist in the records and is invalid.
+
+If the GTID is purged or invalid, follow these steps:
+
+1. Increase binlog retention by configuring `binlog_expire_logs_seconds` in MySQL or through your cloud provider:
```

Comment:
Will be good to note that for managed MySQL services, this is usually configured by the provider, so folks should figure out how their provider allows for binlog retention adjustments.